### DISCUSSION TITLE
Modding idea: Deeper faction customisation

### DISCUSSION THREAD

**Discussion by Alpharius-30-milenium** [FakeFishGames/Barotrauma#12518] [Ideas and suggestions]
In general I want to suggest 2 things: faction affiliated stations and subs.
1. Affiliated stations: It would be great if it will be possible to have option that defines which faction and subfaction can spawn on that certain outpost type. Even in context of doing some kind of vanilla+ mod. We can have stations that always coalition - big, clean stations with good supplies and good store assortment and on other hand we can have separatist stations where you can see more rust and entropy, where people are literally surviving. With existing faction decor system that kind of atmosphere is barely possible. And if talking about complete overhaul mods which brings another universe in game that kind of option also will be pretty useful to prevent some kind of memes when coalition controls station that belongs to guys who want to get rid of it.
2. Affiliated submarines: this kind of thing is good for general mod atmosphere and even may be good for vanilla. You can assign to faction submarines that you want to see on their shipyards, so, for example Winterhalters will belong to coalition and humpback for example will belong to separatists. Or we can go different way and make it so that some models of the submarines are more common on separatists shipyards and others are more common on coalition shipyards.

**Comment by Hazmat616** [comment]
Any sub with husk or clown graffiti can be in those two factions. Humpback would be sold at both.

**Reply by Alpharius-30-milenium** [reply]
I didn't taken to account subfactions in 2-nd case because they did not supposed to have their own shipyards
